.ci/jenkins/Jenkinsfile.bump-up-version

```
@@ -0,0 +1,110 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+@Library('jenkins-pipeline-shared-libraries')_
+
+import org.kie.jenkins.MavenCommand
+
+def repoName = 'kie-benchmarks'
+
+pipeline {
+    agent {
+        label 'kie-rhel8 && !built-in'
+    }
+    tools {
+        maven env.BUILD_MAVEN_TOOL
+        jdk env.BUILD_JDK_TOOL
+    }
+
+    environment {
+        BRANCH_HASH = "${util.generateHash(10)}"
+    }
+
+    options {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+    }
+
+    stages {
+        stage('Initialize') {
+            steps {
+                script {
+                    cleanWs()
+
+                    dir("${repoName}") {
+                        checkoutRepo("${repoName}", "${getTargetBranch()}")
+                    }
+                }
+            }
+        }
+        stage('bump up version') {
+            steps {
+                script {
+                    dir("${repoName}") {
+                        String commitMsg = "bumped up ${getUpdateRepo()} version to next SNAPSHOT"
+                        String localBranch = "update-${getUpdateRepo()}-${env.BRANCH_HASH}"
+                        githubscm.createBranch(localBranch)
+                        if (getUpdateRepo() == 'drools') {
+                            maven.mvnSetVersionProperty('version.org.drools', getDroolsVersion())
+                        }
+                        if (getUpdateRepo() == 'optaplanner') {
+                            maven.mvnSetVersionProperty('version.org.optaplanner', getOptaPlannerVersion())
+                        }
+                        String prLink = commitAndCreatePR(commitMsg, localBranch, "${getTargetBranch()}")
+                        sh "git checkout ${getTargetBranch()}"
+                        mergeAndPush(prLink, "${getTargetBranch()}")
+                        githubscm.removeRemoteBranch('origin', localBranch, getGitAuthorCredsId())
+                    }
+                }
+            }
+        }
+    }
+    post{
+        always{
+            cleanWs()
+        }
+    }
+}
+
+String getDroolsVersion() {
+    return params.DROOLS_VERSION
+}
+
+String getOptaPlannerVersion() {
+    return params.OPTAPLANNER_VERSION
+}
+
+String getUpdateRepo() {
+    return env.REPO_NAME
+}
```

Make this more dynamic.
For example, you could have as an environment varialble a `DEPENDENCY_PROJECT`
and then you would have:

```suggestion
String getDroolsVersion() {
    return params.DROOLS_VERSION
}

String getVersion() {
    return params."${DEPENDENCY_PROJECT.toUpperCase()}_VERSION"
}

String getDependencyProject() {
    return env.DEPENDENCY_PROJECT
}
```

---

could be yes. This is just an example. Adapt it as you wish ;)